src/cromshell/logs/command.py

```diff
@@ -204,7 +211,11 @@ def print_task_logs(
             shardstring = (
                 "" if not sharded else "-shard-" + str(shard_list[i]["shardIndex"])
             )
-            logs = shard_list[i]["backendLogs"]["log"]
```

Comment:
what is the local verison of backend logs

Reply:
Double checked,
For a task executed locally, the metadata doesn't show any log files.
```
  "HelloWorld.HelloWorldTask": [
      {
          "attempt": 1,
          "backend": "Local",
          "backendStatus": "Done",
          "callCaching": {
              "allowResultReuse": false,
              "effectiveCallCachingMode": "CallCachingOff"
          },
          "callRoot": "/cromwell-executions/HelloWorld/ffd951de-154f-4c41-a95f-651073795cb1/call-HelloWorldTask",
          "commandLine": "    set -e\necho 'Hello World!'",
          "end": "2022-10-21T14:36:18.026Z",
          "executionEvents": [
              {
                  "description": "RequestingExecutionToken",
                  "endTime": "2022-10-21T14:36:12.487Z",
                  "startTime": "2022-10-21T14:36:06.811Z"
              },
              {
                  "description": "WaitingForValueStore",
                  "endTime": "2022-10-21T14:36:12.503Z",
                  "startTime": "2022-10-21T14:36:12.487Z"
              },
              {
                  "description": "Pending",
                  "endTime": "2022-10-21T14:36:06.811Z",
                  "startTime": "2022-10-21T14:36:06.782Z"
              },
              {
                  "description": "PreparingJob",
                  "endTime": "2022-10-21T14:36:12.599Z",
                  "startTime": "2022-10-21T14:36:12.503Z"
              },
              {
                  "description": "UpdatingJobStore",
                  "endTime": "2022-10-21T14:36:18.027Z",
                  "startTime": "2022-10-21T14:36:17.059Z"
              },
              {
                  "description": "RunningJob",
                  "endTime": "2022-10-21T14:36:17.059Z",
                  "startTime": "2022-10-21T14:36:12.599Z"
              }
          ],
          "executionStatus": "Done",
          "inputs": {
              "boot_disk_size_gb": null,
              "command_mem": 2048,
              "cpu": null,
              "default_boot_disk_size_gb": 15,
              "default_disk_space_gb": 100,
              "default_ram_mb": 3072,
              "disk_space_gb": null,
              "docker": "frolvlad/alpine-bash",
              "machine_mem": 3072,
              "mem": null,
              "preemptible_attempts": null,
              "use_ssd": false
          },
          "jobId": "238",
          "outputs": {},
          "returnCode": 0,
          "runtimeAttributes": {
              "continueOnReturnCode": "0",
              "failOnStderr": "false",
              "maxRetries": "0"
          },
          "shardIndex": -1,
          "start": "2022-10-21T14:36:06.753Z",
          "stderr": "/cromwell-executions/HelloWorld/ffd951de-154f-4c41-a95f-651073795cb1/call-HelloWorldTask/execution/stderr",
          "stdout": "/cromwell-executions/HelloWorld/ffd951de-154f-4c41-a95f-651073795cb1/call-HelloWorldTask/execution/stdout"
      }
  ]
```